src/bo4e/bo/marktteilnehmer.py

```diff
@@ -41,6 +42,7 @@ class Marktteilnehmer(Geschaeftspartner):
     rollencodetyp: Optional[Rollencodetyp] = None
     #: Sparte des Marktteilnehmers, z.B. Gas oder Strom
     sparte: Optional[Sparte] = None
-
     #: Die 1:1-Kommunikationsadresse des Marktteilnehmers; Diese wird in der Marktkommunikation verwendet.
     makoadresse: Optional[str] = None
```

Comment:
```suggestion
    makoadresse: Optional[list[str]] = None
```
cause it can be an email address or a AS4 endpoint